examples/five_pl_logistic/five_pl_fit.ipynb

```diff
@@ -0,0 +1,934 @@
+      "Requirement already satisfied: python-dateutil>=2.7 in /Users/nickflores_ganymede/Library/Caches/pypoetry/virtualenvs/bio-curve-fit-WuAbQiBo-py3.11/lib/python3.11/site-packages (from matplotlib>=3.7.1->bio-curve-fit) (2.9.0.post0)\r\n",
+      "Requirement already satisfied: pytz>=2020.1 in /Users/nickflores_ganymede/Library/Caches/pypoetry/virtualenvs/bio-curve-fit-WuAbQiBo-py3.11/lib/python3.11/site-packages (from pandas>=1.5.3->bio-curve-fit) (2024.1)\r\n",
+      "Requirement already satisfied: tzdata>=2022.1 in /Users/nickflores_ganymede/Library/Caches/pypoetry/virtualenvs/bio-curve-fit-WuAbQiBo-py3.11/lib/python3.11/site-packages (from pandas>=1.5.3->bio-curve-fit) (2024.1)\r\n",
+      "Requirement already satisfied: joblib>=1.1.1 in /Users/nickflores_ganymede/Library/Caches/pypoetry/virtualenvs/bio-curve-fit-WuAbQiBo-py3.11/lib/python3.11/site-packages (from scikit-learn>=1.2.2->bio-curve-fit) (1.4.2)\r\n",
+      "Requirement already satisfied: threadpoolctl>=2.0.0 in /Users/nickflores_ganymede/Library/Caches/pypoetry/virtualenvs/bio-curve-fit-WuAbQiBo-py3.11/lib/python3.11/site-packages (from scikit-learn>=1.2.2->bio-curve-fit) (3.5.0)\r\n",
+      "Requirement already satisfied: six>=1.5 in /Users/nickflores_ganymede/Library/Caches/pypoetry/virtualenvs/bio-curve-fit-WuAbQiBo-py3.11/lib/python3.11/site-packages (from python-dateutil>=2.7->matplotlib>=3.7.1->bio-curve-fit) (1.16.0)\r\n",
+      "\r\n",
+      "\u001B[1m[\u001B[0m\u001B[34;49mnotice\u001B[0m\u001B[1;39;49m]\u001B[0m\u001B[39;49m A new release of pip is available: \u001B[0m\u001B[31;49m24.1\u001B[0m\u001B[39;49m -> \u001B[0m\u001B[32;49m24.3.1\u001B[0m\r\n",
+      "\u001B[1m[\u001B[0m\u001B[34;49mnotice\u001B[0m\u001B[1;39;49m]\u001B[0m\u001B[39;49m To update, run: \u001B[0m\u001B[32;49mpip install --upgrade pip\u001B[0m\r\n"
+     ]
+    }
+   ],
+   "execution_count": 1
+  },
+  {
+   "cell_type": "code",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-11-20T07:44:52.880521Z",
+     "start_time": "2024-11-20T07:44:50.807560Z"
+    }
+   },
+   "source": [
+    "import pandas as pd\n",
+    "import io\n",
+    "import numpy as np\n",
+    "from matplotlib import pyplot as plt\n",
+    "from bio_curve_fit.logistic import FourPLLogistic, FivePLLogistic\n",
+    "from bio_curve_fit.plotting import plot_standard_curve "
+   ],
+   "outputs": [],
+   "execution_count": 2
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": "# Fitting a Five-Parameter Logistic Model to Create a Standard Curve using `bio-curve-fit`"
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": "For the purposes of demonstration, let's imagine we have 20 samples with a known concentration (standards) of an analyte. We've tested these standards, in triplicate, on an appropriate instrument and recorded the observed response signal. "
```

Comment:
worth putting a sentence or so motivating why the 5PL is going to be useful, otherwise this tutorial might look too much like a clone of the 4PL one